### PR TITLE
TF version bump to resolve v2-nightly, libtpu-nightly runtime issues

### PR DESCRIPTION
**Pull request by yeounoh**
Pinning to the (more) recent TF version that works with `v2-nightly` and `libtpu-nightly`, in preparation for `1.11` release.

### CODE REVIEW

**Comment by JackCaoG** [comment]
If I remember correctly, this is to avoid the `TPUEmbedding` op failure? We still need to use `continuous` to make it work on TPUVM right?

**Comment by yeounoh** [comment]
> If I remember correctly, this is to avoid the `TPUEmbedding` op failure? We still need to use `continuous` to make it work on TPUVM right?

Correct, `TPUEmbedding` and also `devcie access` issues. The `continuation` fix will have to happen outside of this PR: 1) if we modify XLA, re-pin Tensorflow; 2) if we don't modify XLA, then we will set the `no-continuation` flag in `v2-nightly`.

**Comment by JackCaoG** [comment]
Let's wait a couple days and see if we can also fix other tf/libtpu issues before merging. My guess is that change might be in the libtpu end but we might want to update tf to match that.

**Review by JackCaoG — APPROVED** [review]

**Comment by JackCaoG** [comment]
Merge this to unblock cloud build

**Pull request merged**
